File: backend/src/main/java/com/simats/neoomfs/dto/request/RadiologyRequest.java
package com.simats.neoomfs.dto.request;

import lombok.Data;

/**
 * Inbound DTO for radiology data (Step 3 of the wizard).
 */
@Data
public class RadiologyRequest {
    private boolean iopaTaken;
    private String iopaFileUrl;
    private String iopaFindings;

    private boolean opgTaken;
    private String opgFileUrl;
    private String opgFindings;

    private boolean cbctTaken;
    private String cbctFileUrl;
    private String cbctFindings;

    private Double boneDensityHu;
    private String generalRadiologyNotes;
}

File: frontend/app/src/main/java/com/simats/neoomfs/models/RadiologyRequest.kt
package com.simats.neoomfs.models

data class RadiologyRequest(
    val iopaTaken: Boolean,
    val iopaFileUrl: String?,
    val iopaFindings: String? = null,
    val opgTaken: Boolean,
    val opgFileUrl: String?,
    val opgFindings: String? = null,
    val cbctTaken: Boolean,
    val cbctFileUrl: String?,
    val cbctFindings: String? = null,
    val boneDensityHu: Double? = null,
    val generalRadiologyNotes: String? = null
)
